src/services/ResourceBookingService.js

```diff
@@ -228,11 +228,33 @@ deleteResourceBooking.schema = Joi.object().keys({
 /**
  * List resourceBookings
  * @params {Object} criteria the search criteria
+ * @params {Object} options the extra options to control the function
  * @returns {Object} the search result, contain total/page/perPage and result array
  */
-async function searchResourceBookings (criteria) {
+async function searchResourceBookings (criteria, options = { returnAll: false }) {
+  if (criteria.projectIds) {
+    if ((typeof criteria.projectIds) === 'string') {
```

Comment:
Can we directly check for `(typeof criteria.projectIds) === 'string'` removing additional `if (criteria.projectIds) {`?

Reply:
@maxceem Of course we can remove the additional check.